docs/deployment/play-store.mdx

```diff
@@ -88,7 +88,7 @@ As of now, existing apps aren't required to use the AAB format, but you can stil
 
 To opt into app signing, you'll need to upload the app signing key used to sign previous releases of the app. Navigate to Setup -> App integrity, then choose one of the two methods seen in the screenshot below. Once the key has been uploaded, you can enroll in Play App Signing.
 
-![App integrity setup in Google Play Console](https://blog.ionicframework.com/wp-content/uploads/2021/12/existingapps-optin.png)
+![Screenshot highlighting the opt-in options for Play App Signing in the Google Play Console.](https://blog.ionicframework.com/wp-content/uploads/2021/12/existingapps-optin.png "Google Play Console Opt-in to Play App Signing")
```

Comment:
This one also needs to updated to exclude "screenshot". The same goes for the same sentences found in other files.